com.civicrm.hrjobroles/js/src/job-roles/controllers/add-new-role.html

```diff
@@ -214,7 +214,18 @@
             <table class="table">
               <thead>
                 <tr>
-                  <th style="width:40%">Funder</th>
+                  <th style="width:40%">
+                    <div class="row">
```

Comment:
Bootstrap grid inside a table header looks a bit strange. Why would we need such a markup?

Reply:
I need to position the tool icon close to the `Funder` header. Floating the tool icon seems far away. 🤷‍♂️ 
![tool](https://user-images.githubusercontent.com/1507645/48837912-ff559700-ed86-11e8-890c-4154ce192ce0.png)

Can you please suggest a markup?

Reply:
I see. The problem you have probably faced is that `<a>` inside `<th>` becomes a **block** and gets rendered underneath the "Funders" label. I think your solution may not be the best one because:
1) it takes to much markup
2) it renders a grid inside a table heading
3) It is caption length dependent
4) it renders the wrench on a distance from the caption

I would suggest try smth like this:

```html
<th style="width:40%">
  Funder
  <button class="btn btn-link btn-xs"
    ng-click="jobroles.openOptionsEditor('hrjc_funder')">
    <i class="btn btn-link btn-xs fa fa-wrench"></i>
  </button>
</th>
```